### DISCUSSION TITLE
Polish Language

### DISCUSSION THREAD

**Discussion by outof-place** [deepgram/community#717] [Feature Requests]
Dear DeepGram Team,

I have noticed that a significant number of Polish users are utilizing your software. I suggest implementing an automatic Polish language detection feature in your application. This enhancement would greatly facilitate the use of DeepGram for many users in Poland. Adding this feature could not only improve user experience but also increase the popularity of your product among Polish-speaking communities.

Best regards, Filip

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@sercamembert Our [Nova-2 model does support Polish](https://developers.deepgram.com/docs/models-languages-overview#nova-2), can you share more about why you need language detection specfically and what issues you might be having by having to pass in the language parameter of `pl` in your requests?

**Comment by snipd-mikel** [comment]
Hi. We also need to detect wether a given audio file is Polish or not. Are there any plans to add it to the language detection API, now that the nova-2 model already supports it?
